android/trustwalletcore/local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Mon Aug 02 17:42:39 CEST 2021
+sdk.dir=/Users/adam/Library/Android/sdk
```

Comment:
can you revert this local file? also ignore `.settings` folder

Reply:
also please squash all commits into one to make the history cleaner

Reply:
Hm, non-clean merge master added again some local files...  removed

Reply:
Commits will be squashed when merging PR

Reply:
I know, that doesn't make this pr clean (30+ unrelated commits), and the default commit message is a mess

Reply:
try this: 
1. `git reset --soft master`
2. then create a new commit with essential message + force push